src/me/StevenLawson/TotalFreedomMod/TFM_Util.java

```diff
@@ -482,7 +482,7 @@ else if (kicks >= 3)
                 cal.add(Calendar.MINUTE, 1);
                 final Date expires = cal.getTime();
 
-                TFM_Util.bcastMsg(ChatColor.RED + player.getName() + " has been banned for 1 minute.");
```

Comment:
@Wild1145 is right, Red means it is bad, but yellow is a good color :-1: so people might think it is good. Well some people